backstop_data/engine_scripts/puppet/page-objects/crm-page.js

```diff
@@ -98,6 +99,22 @@ module.exports = class CrmPage {
     await Promise.all([
       this.engine.click(selector),
       this.engine.waitForNavigation()
-    ]);      
+    ]);
+  }
+
+  /**
+   * Checks if there are no price fields present on the page or not
+   * @param {String} type - the title of the message to be displayed
+   */
+  async checkIfPriceFieldsAreEmpty (type) {
+    // checks for the messages div - appears if the list is empty.
+    const priceFieldsEmpty = await this.engine.$('#crm-main-content-wrapper .messages');
+    if (priceFieldsEmpty) {
+      console.warn('No Price list present!');
+      console.log(`Taking the "Price ${type} List" Page screenshot..`);
+      return Promise.reject(new Error('list empty'));
```

Comment:
Please add a line break above. We should break the following things
```js
const i = 0;

obj.prop = 1;

func();

if (block) {
  // ...
}

return;
```

Reply:
Done!